Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.7-slim-buster
+
+RUN apt-get update \
+        && apt-get install -y build-essential build-essential python3.7-dev python-igraph
+
+COPY . /pimbook
+WORKDIR "/pimbook"
+
+RUN pip3 install -r requirements.txt
+
+ENTRYPOINT ["python3", "-i"]
```

Comment:
I think if you replace this with ENTRYPOINT ["pytest"] it will run pytest, though it's not clear to me how one would want to view the output. I think you can also have it open bash, connect to it interactively, and run `pytest` from there. Not sure what you had in mind.

Reply:
Oh yeah its only to open it in bash and run tests / code interactively.